android/app/src/main/java/app/organicmaps/bookmarks/data/MapObject.java

```diff
@@ -269,9 +270,9 @@ public String getMetadata(Metadata.MetadataType type)
   }
 
   @NonNull
-  public String getWebsiteUrl(boolean strip)
+  public String getWebsiteUrl(boolean strip, @NonNull Metadata.MetadataType type)
   {
-    final String website = getMetadata(Metadata.MetadataType.FMD_WEBSITE);
+    final String website = Uri.decode(getMetadata(type));
```

Comment:
This decoded URL is not saved anywhere in the Editor, right? And it is clickable, correct?

Reply:
Yes, why ?